pumi-pic-policy-compatibility.md

```diff
@@ -0,0 +1,48 @@
+|**M2.** Provide a comprehensive test suite for correctness of installation verification. |Full| Supports ctest
+|**M3.** Employ user-provided MPI communicator (no MPI_COMM_WORLD). Don't assume a full MPI 3 implementation without checking. Provide an option to prevent any changes to MPI error-handling if it is changed by default. |Full| None. |
+|**M4.** Give best effort at portability to key architectures (standard Linux distributions, GNU, Clang, vendor compilers, and target machines at ALCF, NERSC, OLCF). |Full| None. |
+|**M5.** Provide a documented, reliable way to contact the development team. |Full| Developers can be contacted through github repository or as specified in the SUPPORT.md file. |
+|**M6.** Respect system resources and settings made by other previously called packages (e.g. signal handling). |Full| None. |
+|**M7.** Come with an open source (BSD style) license. |Full| Using BSD 3-Clause License. |
+|**M8.** Provide a runtime API to return the current version number of the software. |Full| Version can be retrieved from `pumipic_version()` function. |
+|**M9.** Use a limited and well-defined symbol, macro, library, and include file name space. |Full| None. |
+|**M10.** Provide a publicly available repository. |Full| Pumi-Pic is hosted in a public repository at: https://github.com/SCOREC/pumi-pic |
+|**M11.** Have no hardwired print or IO statements that cannot be turned off. |Full| IO statements can be disabled with a compiler flag `PUMIPIC_PRINT_ENABLED` or at runtime using optional spdlog |
+|**M12.** For external dependencies, allow installing, building, and linking against an outside copy of external software. |Full| None. |
+|**M13.** Install headers and libraries under \<prefix\>/include and \<prefix\>/lib. |Full| This can be done with the standard configuration variable in CMake. |
+|**M14.** Be buildable using 64 bit pointers. 32 bit is optional. |Full| Package supports 64 bit. |
+|**M15.** All xSDK compatibility changes should be sustainable. |Full| None. |
+|**M16.** Have a debug build option. |Full| None. |
+|**M17.** Each xSDK member package should have sufficient documentation to support use and further development.  |Full| Contains installation instrutions and a programming guide in the github wiki: https://github.com/SCOREC/pumi-pic/wiki. Also maintains doxygen documentaion on github pages: https://scorec.github.io/pumi-pic/ |
+
+### Recommended Policies
+
+| Policy                 |Support| Notes                   |
+|------------------------|-------|-------------------------|
+|**R1.** At least one validation (smoke) test that can be invoked through the Spack package. |Full| None. |
+|**R2.** Possible to run test suite under valgrind in order to test for memory corruption issues. |Full| None. |
+|**R3.** Adopt and document consistent system for error conditions/exceptions. |Partial| No hardwired IO statements. |
```

Comment:
What is the relationship between IO statements and error conditions/exceptions?

Reply:
I rewrote to make the relationship more clear. The reason is in the full description:
https://github.com/xsdk-project/xsdk-community-policies/blob/master/package_policies/R3.md